cmake/modules/FindPySpark.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#
+# Try to find pyspark
+# Once done this will define
+#
+#  PYSPARK_FOUND - system has pyspark and its dependencies are met
+#
+
+message(STATUS "Looking for pyspark dependency: Java")
+find_package(Java 1.8 COMPONENTS Runtime)
+
+if (Java_Runtime_FOUND)
+    message(STATUS "Found Java ${Java_JAVA_EXECUTABLE}")
+    message(STATUS "Java version ${Java_VERSION_STRING}")
+    message(STATUS "Looking for pyspark dependency: py4j")
+
+    execute_process(
+        COMMAND ${PYTHON_EXECUTABLE_Development_Main} -c "import py4j"
+        RESULT_VARIABLE _PY4J_SEARCH_SUCCESS
+        OUTPUT_VARIABLE _PY4J_VALUES_OUTPUT
+        ERROR_VARIABLE _PY4J_ERROR_VALUE
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+
+    if(${_PY4J_SEARCH_SUCCESS} EQUAL 0)
+        message(STATUS "Package 'py4j' has been succesfully imported with ${PYTHON_EXECUTABLE_Development_Main}")
+        message(STATUS "Import python package: pyspark")
+
+        execute_process(
+            COMMAND ${PYTHON_EXECUTABLE_Development_Main} -c "import pyspark; print(pyspark.__version__)"
```

Comment:
we are checking pyspark only for PYTHON_EXECUTABLE_Development_Main, right? We need to mention it somewhere (e.g. README or notes)...

Reply:
It will be mentioned in the FindPySpark module and also in the relnotes :+1: